examples/pitr/main.tf

```diff
@@ -0,0 +1,29 @@
+##############################################################################
+# Resource Group
+##############################################################################
+
+module "resource_group" {
+  source = "git::https://github.com/terraform-ibm-modules/terraform-ibm-resource-group.git?ref=v1.0.5"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+locals {
+  permanent_resources = yamldecode(file("../../common-dev-assets/common-go-assets/common-permanent-resources.yaml"))
+}
+
+data "ibm_database_point_in_time_recovery" "database_pitr" {
+  deployment_id = local.permanent_resources["postgresqlCrn"]
+}
```

Comment:
this should be done in the go test, not the example. The example just need to have a variable where any CRN can be passed so its not specific to our account

Reply:
see example https://github.com/terraform-ibm-modules/terraform-ibm-cos/blob/main/tests/pr_test.go#L26

Reply:
implemented.